TrackletGraph.py

```diff
@@ -274,9 +294,12 @@ def get_proc_dict_from_config(fname_pconfig):
         for i,line_proc in enumerate(file_proc):
             # Processing module type
             proc_type = line_proc.split(':')[0].strip()
-            # temperary hack: DiskMatchCalculator-->MatchCalculator
+            # temporary hack: DiskMatchCalculator-->MatchCalculator
```

Comment:
Why is this hack necessary? The file emData/LUTs/processingmodules.dat doesn't contain the string DIskMatchCalculator.

Reply:
I suspect this was needed for an older version of the wiring. It was added by Tao, here in this commit:
https://github.com/cms-L1TK/project_generation_scripts/commit/2cba54894bbcfcd7f33981d4dcb4fb90e84eead5#diff-67bcb68c62e646510a9286c8eed7bf88f48295cb2252257e762e4e38a6d6c3c5R113-R115
I was just fixing the spelling error since I had copied the comment.

Reply:
Try removing it. If it still works, delete it ...

Reply:
For the sake of consistency, I removed all of the hacks for the DiskMatchCalculator throughout (excluding the Tracklet1.0 directory), and the scripts still work and produce identical results with the current wiring.